### PR TITLE
Create - 2023-11-10-Raft.md

### PR DESCRIPTION
**Pull request by aholman0725**

### CODE REVIEW

File: content/attacks/posts/2023-11-10-RaftExploit [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Timeline
+
+- **November 10, 2023:** Raft co-funder David Garai confirmed in a post on X (formerly Twitter) that the platform was targeted by an [attack](https://twitter.com/davgarai/status/1723065357445775507?s=20).
+- **November 10, 2023:** 0xngmi discusses the hacker's [losses](https://twitter.com/0xngmi/status/1723073285263380924?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1723073285263380924%7Ctwgr%5Ea11731dce5d8111bfcb51bbcaf2447980e7e2d00%7Ctwcon%5Es1_&ref_url=https%3A%2F%2Fwww.coindesk.com%2Ftech%2F2023%2F11%2F10%2Fdefi-platform-raft-suffers-33m-exploit-but-hacker-likely-takes-a-loss-on-the-attack%2F)
+- **November 10, 2023:** Coindesk reports the [incident](https://www.coindesk.com/tech/2023/11/10/defi-platform-raft-suffers-33m-exploit-but-hacker-likely-takes-a-loss-on-the-attack/)
+
+## Security Failure Causes
+
+The exploiter minted R (which was then sold to drain AMM liquidity), and also managed to withdraw collateral at the same time.
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-RaftExploit:31]: how was teh exploiter able to mint the tokens?

File: content/attacks/posts/2023-11-10-RaftExploit [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+date: 2023-11-10  
+target-entities: Raft R
+entity-types: Decentralized finance (DeFi) platform  
+attack-types: Liquidity token exploit
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-RaftExploit:5]: use standard attack types from list in issue

File: content/attacks/posts/2023-11-10-RaftExploit [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+title: “Raft Suffers $3.3M Exploit That Drove Down Stablecoin 50%, but Hacker Likely Lost Money on Attack”
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-RaftExploit:6]: simplify title: ie "Raft Suffers 3.3M Exploit"

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

**Comment by aholman0725** [comment]
Requested changes complete. Thanks for clear instructions.

File: content/attacks/posts/2023-11-10-RaftExploit [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+The identity of the attacker is currently unknown.
+
+## Losses
+
+Raft R lost 1,577 [ETH](https://blockchain.news/news/defi-platform-raft-compromisedloses-33-million-in-ether#:~:text=resulted%20in%20the%20theft%20of%20around%20%243.3%20million%20worth%20of%20Ether%20(ETH)) during the attack, worth 3,300,000 USD at the time of the attack.
+
+## Timeline
+
+-November 10, 2023/14:49: Raft co-funder David Garai confirmed in a post on X (formerly Twitter) that the platform was targeted by an [attack](https://twitter.com/davgarai/status/1723065357445775507?s=20).
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-RaftExploit:23]: small edit: follow timing syntax convention from other articles

Comment (aholman0725) [reply content/attacks/posts/2023-11-10-RaftExploit]: Timeline format updated.

File: content/attacks/posts/2023-11-10-RaftExploit [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+The identity of the attacker is currently unknown.
+
+## Losses
+
+Raft R lost 1,577 [ETH](https://blockchain.news/news/defi-platform-raft-compromisedloses-33-million-in-ether#:~:text=resulted%20in%20the%20theft%20of%20around%20%243.3%20million%20worth%20of%20Ether%20(ETH)) during the attack, worth 3,300,000 USD at the time of the attack.
+
+## Timeline
+
+-**November 10, 2023, 14:49 PM:** Raft co-funder David Garai confirmed in a post on X (formerly Twitter) that the platform was targeted by an [attack](https://twitter.com/davgarai/status/1723065357445775507?s=20).
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-RaftExploit:23]: sorry, just make sure to convert times to UTC and then add "UTC"

Comment (aholman0725) [reply content/attacks/posts/2023-11-10-RaftExploit:23]: Will do. 

Is there a simple way to figure out what time zone an article is posted? A lot of the time it'll just say something like "12 hours ago".

**Review by jhirschkorn — APPROVED** [review]

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]
1. fix file name
2. fix links
3. add details about the attack

**Comment by aholman0725** [comment]
> 1. fix file name
> 2. fix links
> 3. add details about the attack

Could you please expand on point number two.

**Comment by evgenydmitriev** [comment]
> > 1. fix file name
> > 2. fix links
> > 3. add details about the attack
> 
> Could you please expand on point number two.

Remove all optional [URL components](https://en.wikipedia.org/wiki/URL#Syntax) (?` and `#` and everything that follows)

File: content/attacks/posts/2023-11-10-RaftExploit (hunk 5 above)
Comment (jhirschkorn) [reply content/attacks/posts/2023-11-10-RaftExploit]: depends on how they timestamp it

File: content/attacks/posts/2023-11-10-Raft [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+The identity of the attacker is currently unknown.
+
+## Losses
+
+Raft R lost 1,577 [ETH](https://blockchain.news/news/defi-platform-raft-compromisedloses-33-million-in-ether) during the attack, worth 3,300,000 USD at the time of the attack.
+
+## Timeline
+
+**November 10, 2023, 19:18 UTC PM:** Raft posts a [tweet](https://twitter.com/raft_fi/status/1723057566664548623) stating they are investigating a prospective safety issue. 
+**November 10, 2023, 19:49 UTCpm:** Raft co-funder David Garai confirmed in a post on X (formerly Twitter) that the platform was targeted by an [attack.](https://twitter.com/davgarai/status/1723065357445775507)
```
Comment (jhirschkorn) [inline content/attacks/posts/2023-11-10-Raft:24]: if it's 24 hr time you dont need am/pm

**Comment by aholman0725** [comment]
@evgenydmitriev requested changes made. Please let me know if I missed anything.

**Comment by aholman0725** [comment]
@evgenydmitriev I pretty much started from scratch for this article.

**Comment by aholman0725** [comment]
@JediFaust I look forward to your feedback.

**Review by evgenydmitriev — APPROVED** [review]

**Comment by evgenydmitriev** [comment]
@aholman0725, please pay attention to the file naming convention. It is important for the wiki functioning. Also, don't forget to [explicitly re-request reviews](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review#:~:text=re%2Drequest%20review%20from%20the%20same%20reviewer) if you need us to give it another look.

**Comment by aholman0725** [comment]
@evgenydmitriev Thanks for the example for re-requests. I re-requested a few pull requests last week, but you are the listed reviewer. Not sure if you want to add Omar since he's helping. Thanks again.

**Pull request merged**